client/src/modules/accounts/accounts.routes.js

```diff
@@ -44,6 +55,15 @@ function accountsModal($modal) {
   });
 }
 
+function importAccountsModal($modal) {
+  $modal.open({
+    keyboard : false,
```

Comment:
Just FYI - the `keyboard: false` and `backdrop: 'static'` are set by default [in app.js](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/app.js#L208).